### DISCUSSION TITLE
KeyBoardAccessoryView not show on IOS but show on android

### DISCUSSION THREAD

**Discussion by TruongMinhThuan** [wix/react-native-ui-lib#1968] [Ideas]
![Screen Shot 2022-04-12 at 13 36 09](https://user-images.githubusercontent.com/89587992/162896126-9352e33e-9ba5-4362-bdd1-c2d276996d66.png)

**Comment by Inbal-Tish** [comment]
@TruongMinhThuan Hi. I'm looking into it. What version of the library are you running?

**Comment by Inbal-Tish** [comment]
Also, add a code snippet for your input and the KayboardAccessoryView usages
